config/core/test-configs.yaml

```diff
@@ -2171,6 +2177,24 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-uboot
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-lkdtm
+      - kselftest-rtc
+      - kselftest-seccomp
+      - ltp-crypto
+      - ltp-fcntl-locktests
+      - ltp-ipc
+      # ltp-mm - runs system out of memory
+      - ltp-pty
+      - ltp-timers
```

Comment:
OK with some failing cases but that's not specific to Juno: https://staging.kernelci.org/test/plan/id/6265a0b02ab8578665178696/